public/js/test/fixtures.json

```diff
@@ -4,12 +4,14 @@
       "fooSourceActor": {
         "actor": "fooSourceActor",
         "url": "http://example.com/foo/foo.js",
-        "filename": "foo.js"
+        "filename": "foo.js",
+        "pathname": "foo/foo.js"
       },
       "barSourceActor": {
         "actor": "barSourceActor",
         "url": "http://example.com/bar/bar.js",
-        "filename": "bar.js"
+        "filename": "bar.js",
+        "pathname": "bar/bar.js"
```

Comment:
i feel guilty adding filename and pathname to the fixtures. should really be a query